docs/Introductie van Open Source Software en OSPOs.md

```diff
@@ -0,0 +1,59 @@
+# Introductie van Open Source Sofware en OSPOs
+
+## 1. Open Source Software 
+
+### 1.1 Open Source Software definitie
+
+Open source sofware (OSS) is sofware waarvan de broncode openbaar is. De broncode is het geheel van
+voor mensen leesbare programma-instructies die de programmeur (de maker) geschreven heef.
+Daardoor is te zien hoe het programma in de basis werkt. 
+
+### 1.2 Voordelen van Open source Software 
+
+Over het algemeen is open source een manier van werken waarbij auteurs de broncode vrij beschikbaar
+stellen voor anderen die de code willen bekijken, kopiëren, ervan leren, aanpassen of delen.
+Wat deze manier van werken betekent voor een organisatie, hangt af van de context van die
+organisatie.
+
+Voor een commerciële sofwareontwikkelaar biedt open source toegang tot state-of-the-art
+technologie en kan het diens innovatie- en ontwikkelprocessen versnellen en licentiekosten
+besparen. Het vergroot de hulpmiddelen waarover de organisatie kan beschikken.
+In de context van het openbaar bestuur zijn de voordelen breder. Open source is een strategisch
+beleidsinstrument om maatschappelijke waarde te realiseren.
+
+Publieke waarden
+• Een belangrijk voordeel van een openbare broncode is dat de werking van de sofware en de functies
+die daarmee worden uitgevoerd voor iedereen inzichtelijk zijn. Dat inzicht geef vertrouwen in de
+werking en past in het streven naar een transparante overheid.
+• Open source biedt vele mogelijkheden om op nieuwe manieren samen te werken met ketenpartners en medeoverheden. Het draagt daarmee ook bij aan efectief en efciënt openbaar bestuur.
+• Open source kan verder de mogelijkheden tot samenwerking met inwoners en ondernemers
+vergroten. En daarmee de participatie van de samenleving in het openbaar bestuur versterken,
+en de legitimiteit en het draagvlak voor te maken keuzen in beleid en uitvoering vergroten.
+
+Veiligheid
+• Openbaarheid van de broncode biedt derden de kans om de sofwarekwaliteit te toetsen en op
+fouten of risico’s te onderzoeken. Dit kan bijdragen aan de veiligheid en betrouwbaarheid
+(veel-ogen-principe) van de sofware.
+• Derden kunnen verbetervoorstellen doen.
+
+Efciency en innovatie
```

Comment:
```suggestion
Efficiency en innovatie
```

Reply:
Laten we het gewoon Nederlands houden: Efficiëntie